core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -538,7 +538,7 @@ public void addColumn(
         commit();
 
         long columnNameTxn = getTxn();
-        LOG.info().$("adding column '").utf8(columnName).$('[').$(ColumnType.nameOf(columnType)).$("], columnName txn ").$(columnNameTxn).$(" to ").$substr(pathSize, path).$();
+        LOG.info().$("adding column '").utf8(columnName).$('[').$(ColumnType.nameOf(columnType)).$("], columnName txn ").$(columnNameTxn).$(" to ").$substr(pathRootSize, path).$();
```

Comment:
nit: would be nice to make path logging consistent, e.g. in `addIndex()` it's `.$(path)`, not `.$substr(pathRootSize, path)`

Reply:
I changed the place you mentioned and a few more. I keep full path on partition manipulation and error logging